ARMmbed/meta-mbl
# python3: Remove additional files used to set up the obsolete test environment

For [IOTMBL-936 Remove recipes for packages that are installed in the
test environment](https://jira.arm.com/browse/IOTMBL-963)

Tests:
* Ensure the mbl-console-image builds successfully (in progress see Jenkins)
* Ensure the developer mbl-console-image-test builds successfully (in progress see Jenkins)

Jenkins Job:
http://jenkins.mbed-linux.arm.com/job/hk-test-job-1/16/
